### PR TITLE
Address PR283 review comments

### PR DESCRIPTION
**Pull request by stsci-hack**
This PR will implement changes identified during the code review of PR #283 which were deferred and documented in #295 .  Multiple commits will be needed to resolve all the issues identified in #295.

### CODE REVIEW

File: tests/hla/base_test.py [hunk 1]
```diff
@@ -343,22 +343,16 @@ def print_wcs(self, title, wcs):
...
         """
-        Read the image from a fits file
+        Read the image from first extension of a fits file
         """
-        hdu = fits.open(filename)
-
-        image = hdu[1].data
-        hdu.close()
-        return image
+        return fits.getdata(filename,ext=1)
```
Comment (mcara) [inline tests/hla/base_test.py]:
```suggestion
        return fits.getdata(filename, ext=1)
```

File: tests/hla/base_test.py [hunk 2]
```diff
@@ -343,22 +343,16 @@ def print_wcs(self, title, wcs):
...
     def read_wcs(self, filename):
         """
         Read the wcs of a fits file
         """
-        hdu = fits.open(filename)
-
-        wcs = stwcs.wcsutil.HSTWCS(hdu, 1)
-        hdu.close()
+        from fits.open(filename) as hdu:
+            wcs = stwcs.wcsutil.HSTWCS(hdu, 1)
```
Comment (mcara) [inline tests/hla/base_test.py]: I would just replace this with:
```python
wcs = stwcs.wcsutil.HSTWCS(filename, 1)
```

Comment (mcara) [reply tests/hla/base_test.py]: and delete `from fits.open(filename) as hdu:` line above

**Comment by mcara** [comment]
Also, I would suggest the following change to: https://github.com/spacetelescope/drizzlepac/blob/bdadf13688e47a18d9ae0882d8a19207343d278f/tests/hla/base_test.py#L149-L161

```
if isinstance(output_truth, tuple):
    output_spec = {'files':(output_file, output_truth[0]),
                   'pars': output_truth[1]}    
else:
    if output_hdus:
        output_spec = (output_file, output_truth, output_hdus)
    else:
        output_spec = (output_file, output_truth)
```

File: tests/hla/base_test.py (hunk 2 above)
Comment (jhunkeler) [reply tests/hla/base_test.py]: ^ `from` should be be `with` (and/or these lines can removed as @mcara suggested).

**Comment by stsci-hack** [comment]
All issues addressed in #339.

**Pull request closed without merging**